android-kotlin/app/src/main/java/com/mongodb/app/ItemAdapter.kt

```diff
@@ -32,26 +35,44 @@ internal class ItemAdapter(data: OrderedRealmCollection<Item>, private val confi
         val obj: Item? = getItem(position)
         holder.id = obj?._id
         holder.name.text = obj?.name
+        holder.checkbox.isChecked = obj?.checked?:false
+        holder.checkbox.setOnClickListener { onCheckboxClicked(holder) }
+        holder.menu.setOnClickListener { onMenuClicked(holder) }
```

Comment:
I'm not sure if one-lining these like this is bad practice or not, it looks nicer to me but I'm open to changing it

Reply:
That's fine.